src/figuresClasses.ts

```diff
@@ -1,19 +1,109 @@
-export interface Figure {
 
+export interface Figure {
+  color: 'red' | 'green' | 'blue';
+  shape: string;
+  radius?: number;
+  getArea: () => number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  color: 'red' | 'green' | 'blue';
```

Comment:
Create separate type for all available colors and don't repeat them in your code